### PR TITLE
[17.0][MIG] crm_timesheet: Migration to 17.0

### PR DESCRIPTION
**Pull request by RogerSans**
Standard migration
T-6446

@HaraldPanten 
@ValentinVinagre 
@luis-ron

### CODE REVIEW

**Review by luis-ron — APPROVED** [review]
Functional Review: LGTM 👍🏻

**Comment by RogerSans** [comment]
@manuelregidor Can you review this pls?

File: crm_timesheet/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Copyright 2019 Tecnativa - Jairo Llopis
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "CRM Timesheet",
+    "category": "Customer Relationship Management",
+    "version": "17.0.1.0.0",
+    "depends": ["crm", "project_timesheet_time_control", "sale_timesheet"],
```
Comment (manuelregidor) [inline crm_timesheet/__manifest__.py]: Why is the sale_timesheet dependency necessary in this version? Thank you

Comment (RogerSans) [reply crm_timesheet/__manifest__.py]: Without this dependency, the tests fail because they do not find the billing_type field.

Comment (manuelregidor) [reply crm_timesheet/__manifest__.py]: I've just tried locally and you don't need that dependency. You have added the billing_type field when creating project1 in test and I don't think that you should have done that (note that the field is not mentioned in v16 test file).

Comment (RogerSans) [reply crm_timesheet/__manifest__.py]: Deleted

**Review by manuelregidor — APPROVED** [review]
Technical review. LGTM

**Review by pedrobaeza — APPROVED** [review]
/ocabot migration crm_timesheet
/ocabot merge nobump

**Pull request merged**